Content.Shared/Contraband/ContrabandSystem.cs

```diff
@@ -55,10 +57,23 @@ private void OnExamined(Entity<ContrabandComponent> ent, ref ExaminedEvent args)
         using (args.PushGroup(nameof(ContrabandComponent)))
         {
             var severity = _proto.Index(ent.Comp.Severity);
+            var jobList = new List<string>();
             if (severity.ShowDepartments && ent.Comp is { AllowedDepartments: not null })
```

Comment:
might make sense to rename the ShowDepatments DataField now. Also mention in the field's summary that it handles jobs as well.